File: 代码/kyzh_parent/kyzh_manager_web/src/main/webapp/js/service/articleContentService.js
//服务层
app.service('articleContentService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../articleContent/findAll.do');		
	}
	//分页 
	this.findPage=function(pageNo,pageSize,searchEntity){
		return $http.post('../articleContent/findPage.do?pageNo='+pageNo+'&pageSize='+pageSize, searchEntity);
		//return $http.post('../testdata.json'); //前端测试效果的json文件
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../articleContent/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../articleContent/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../articleContent/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../articleContent/delete.do?ids='+ids);
	}  	
});

File: 代码/kyzh_parent/kyzh_article_interface/src/main/java/com/kyzh/article/service/ArticleContentService.java
package com.kyzh.article.service;
import java.util.List;
import com.kyzh.pojo.TbArticleContent;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ArticleContentService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbArticleContent> findAll();
	
	
	/**
	 * 增加
	*/
	public void add(TbArticleContent articleContent);
	
	
	/**
	 * 修改
	 */
	public void update(TbArticleContent articleContent);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbArticleContent findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页+模糊搜索
	 * 参数一模糊搜索条件
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findAll(TbArticleContent articleContent, int pageNum, int pageSize);
	
}
